### ISSUE TITLE
Rcpp::compileAttributes(): RcppExports.R difference between version 0.12.10 and 0.12.11

### ISSUE THREAD

**Issue by HannahVMeyer** [RcppCore/Rcpp#710]
Version: 0.12.10
```
# Generated by using Rcpp::compileAttributes() -> do not edit by hand
# Generator token: 10BE3573-1514-4C36-9D1C-5A225CD40393

#'Scan file for specific line numbers
#' 
#' @param filename /path/to/chromosomefile [string]
#' @param lines vector of line numbers [integer] to be read
#' @param sep [string] end-of-line delimiter
read_lines <- function(filename, lines, sep = '\n') {
    .Call('PhenotypeSimulator_read_lines', PACKAGE = 'PhenotypeSimulator', filename, lines, sep)
}
```

Version: 0.12.11
```
read_lines <- function(filename, lines, sep = '\n') {
    .Call(PhenotypeSimulator_read_lines, filename, lines, sep)
}
```

**Comment by eddelbuettel** [comment]
Do you have a source file for a _minimally reproducible example_ ? 

My packages with with doxygen comments all work.

As for the different `.Call()` "look-and-feel": that is all due to changed with R 3.4.0.

**Comment by HannahVMeyer** [comment]
Sorry, I am just editing the comment, accidentally sent it before I was finished.

**Comment by eddelbuettel** [comment]
Wild guess:  can you add an extra space in [this line](https://github.com/HannahVMeyer/PhenotypeSimulator/blob/master/src/read_lines.cpp#L8)  so that it looks like this instead:

```
//' Scan file for specific line numbers
```

**Comment by eddelbuettel** [comment]
That *was* the issue:

```
# Generated by using Rcpp::compileAttributes() -> do not edit by hand
# Generator token: 10BE3573-1514-4C36-9D1C-5A225CD40393

#' Scan file for specific line numbers
#' 
#' @param filename /path/to/chromosomefile [string]
#' @param lines vector of line numbers [integer] to be read
#' @param sep [string] end-of-line delimiter
read_lines <- function(filename, lines, sep = '\n') {
    .Call(PhenotypeSimulator_read_lines, filename, lines, sep)
}
```

Also, you can now delete `src/PhenotypeSimulator_init.c`, its content will be part of `src/RcppExports.cpp`.

**Comment by eddelbuettel** [comment]
It has to do with the function registration.  You had a file `src/PhenotypeSimulator_init.c`:  remove that, then run `compileAttributes()` (or use whichever build tool you use that runs it, be it RStudio or devtools or whatever) and have a new `src/RcppExports.cpp` generated.  It will work with Rcpp 0.12.11 (or, in very worst case, 0.12.11.1; see the two most recently closed issue tickets).

**Comment by HannahVMeyer** [comment]
Great, removing the src/PhenotypeSimulator_init.c (which I included after upgrading to R 3.4.0 based on this [issue](https://github.com/RcppCore/Rcpp/issues/636)) and re-compiling with Rcpp 0.12.11  (via devtools) did the trick.  Thank you very much for your help.
